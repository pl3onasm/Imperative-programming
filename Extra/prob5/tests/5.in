10896 10999