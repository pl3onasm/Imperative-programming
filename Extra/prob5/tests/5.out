10965: 10101011010101
